src/views/Players/Players.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { fetchPlayers } from '../../services/fetchplayers';
+import { Link } from 'react-router-dom';
+import './Players.css';
+
+export default function Players() {
+  const [players, setPlayers] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchPlayers();
+
+      setPlayers(data);
+    };
+    fetchData();
+  }, []);
+  return (
+    <div className="players">
+      <h2>Players</h2>
+      {players.map((player) => (
+        <div key={player.id} {...player}>
```

Comment:
Within the div, we won't need to spread the players state, as we are not passing down any props here.